python/paddle/distributed/launch.py

```diff
@@ -117,33 +71,128 @@ def parse_args():
 PADDLE_TRAINER_ENDPOINTS
 POD_IP (current node ip address, not needed for local training)
 ''')
+
+    # Optional arguments for the launch helper
     parser.add_argument(
-        '--gpus',
+        "--node_ips",
+        type=str,
+        default="127.0.0.1",
+        help="Paddle trainer ips, such as 192.168.0.16,192.168.0.17..")
+
+    parser.add_argument(
+        "--node_id",
         type=int,
-        default=8,
-        help='start number of processes for every gpu')
+        default=0,
+        help="The trainer id of the node for multi-node distributed "
+        "training")
+
+    parser.add_argument(
+        "--print_config",
+        type=bool,
+        default=True,
+        help="Print the config or not")
+
     parser.add_argument(
-        '--log_dir',
+        "--selected_gpus",
         type=str,
-        default="mylog",
-        help='directory to put logs per process.')
```

Comment:
more comments are required for describing the actions if this argument is not specified.

Reply:
Done.Thanks!